Omulosi/iReporter
# #162371370  Configure Procfile for deployment to Heroku

#### What does this PR do?
Configures the Procfile file
#### Description of Task to be completed?
Add deployment setting to the Procfile to enable deployment of the application to Heroku
#### How should this be manually tested?
Use Postman to confirm that the endpoints at the root i,e, `https://iwhistler.herokuapp.com/<endpoint-link>`
work as expected
#### What are the relevant pivotal tracker stories?
#162371370
